.npmignore

```diff
@@ -1,3 +1,5 @@
 **/__mocks__/**
 **/__tests__/**
 src
+tsconfig.json
+tsconfig.tsbuildinfo
```

Comment:
could you ensure a trailing newline in these files?

Reply:
Done.